Backend/Core/Interfaces/CustomService/IEmailService.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Core.Entities;
+using System.Threading.Tasks;
+
+namespace Core.Interfaces.CustomService
+{
+    public interface IEmailService
+    {
+        Task SendConfirmationEmailAsync(User user, string callbackUrl);
+        Task SendEmailAsync(string email, string subject, string message);
```

Comment:
you use it only in the EmailService which implements this interface, so delete it (the 9-nd line) & make the method private